### ISSUE TITLE
`exa --tree` with `--git-ignore`

### ISSUE THREAD

**Issue by anishmittal2020** [ogham/exa#569]
When we execute, `exa --tree` with  `--git-ignore`. Paths mentioned in `.gitignored` are not respected.

```
#.gitignore
/**/Unsync/**
```

```
$ exa --tree --git-ignore
.
├── a.pdf
├── B
│  ├── a.pdf
│  ├── b.md
│  └── Unsync
│     ├── a.pdf
│     └── b.md
└── b.md
```

**Comment by kdabir** [comment]
Even I am facing the same problem. The same issue affect `--recurse` option as well.

**Comment by hrlmartins** [comment]
Can you try remove the first `/` from the line of the `.gitignore` to see if it works.

**Comment by alexilyaev** [comment]
Had the same issue with nested `node_modules` folder.

With `exa`, this only ignores the top level `node_modules`, but in git it actually ignores it anywhere in the tree.

```
#.gitignore
node_modules
```

Adding this works for `exa`, but we shouldn't be needing to do this:

```
#.gitignore
node_modules
*/node_modules
```

**Comment by whitelynx** [comment]
I'm also having the same issue, specifically with `__pycache__` folders and `.pyc` files. My `.gitignore` has the lines:
```
__pycache__/
*.py[cod]
```
which correctly make `git` ignore all nested `__pycache__` and `.pyc` files, but `exa--tree --git-ignore` still shows them.

**Comment by pmensik** [comment]
Is there any progress with this issue? I can confirm that `exa -T --git-ignore` will display contents of the `target` directory when [this](https://github.com/substrate-developer-hub/substrate-node-template] project is built.

**Comment by ariasuni** [comment]
I can’t reproduce. I believe this was fixed by #653 and that it should work correctly if you’re using exa 0.10.0.
